src/file_organizer/tui/status.py

```
@@ -0,0 +1,22 @@
+
+class StatusMixin:
+    """Mixin for views that update the app-level status bar."""
+
+    def _set_status(self, message: str) -> None:
+        """Update the app status bar if one is mounted."""
+        try:
+            from file_organizer.tui.app import StatusBar
+
+            app = cast(Any, self).app
+            app.query_one(StatusBar).set_status(message)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)